src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Color {
+    RED, YELLOW, BLUE, WHITE, BLACK;
```

Comment:
When defining an enum like the one you provided, it is considered a best practice to write the enum constants vertically, each on its own line. This makes the code more readable and helps maintain a consistent coding style.